conjureup/controllers/steps/gui.py

```diff
@@ -51,6 +51,14 @@ def get_result(self, future):
         app.log.debug("Storing step result for: {}={}".format(
             step_model.title, step_model.result))
         self.results[step_model.title] = step_model.result
+        if len(self.view.steps) == 0:
+            app.log.debug(
+                "End of step list waiting for last step to complete "
+                "then rendering summary.")
+
+            async.submit(partial(self.finish, None, None, done=True),
```

Comment:
If no steps remain and the last step returns quickly (less than a second) it will seem like it immediately switches to the status screen. We can either put a second delay here or in the spells that may return quickly.